cloudhub-2/modules/ROOT/pages/ch2-patch-updates.adoc

```diff
@@ -21,6 +21,152 @@ When an update is available, CloudHub 2.0 includes it in Production environments
 
 xref:cloudhub::vpn-maintenance.adoc[Anypoint VPNs] used in CloudHub 2.0 are upgraded continuously. Whenever a new update is available, the VPN is patched within the service-level agreement. If the Anypoint VPN is not setup as high availability or if the non-high-availability VPN is not set up with two tunnels, downtime is expected.
 
+== Opt-in Early Infrastructure Upgrades (Optional)
+
+You can opt into early infrastructure upgrades to be applied earlier than the scheduled patching process, between the Release Available and Auto-Update dates stated in the xref:release-notes::cloudhub-2/cloudhub-2-runtimes-release-notes.adoc[monthly updates table]. In this way, you can choose a time that better suits your organization's needs. 
+
+On average, this upgrade takes between thirty to sixty minutes to complete. If your upgrade is queued, it might take longer than thirty minutes. The impact of the upgrade process on running applications is consistent with the auto patching behavior described in the previous sections.
```

Comment:
```suggestion
On average, this upgrade takes between thirty to sixty minutes to complete. If your upgrade is queued, it can take longer than thirty minutes. The impact of the upgrade process on running applications is consistent with the auto-patching behavior described in the previous sections.
```